notebooks/tm/hospitals-isochrones/01-generate-health-isochrones.ipynb

```diff
@@ -0,0 +1,538 @@
+{
```

Comment:
<p>Rename header to "Define Mapbox API functions"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thinkingmachines/lacuna-fund-climate-health/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>